app/assets/stylesheets/components/_avatar.scss

```diff
@@ -7,6 +7,12 @@
   border-radius: 1rem;
 }
 
+.avatar-small {
```

Comment:
please rename this class. it is similar to  bootstrap avatar-sm and it might even overwrite it.